azext_iot/iothub/commands_state.py

```diff
@@ -0,0 +1,52 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azext_iot.iothub.providers.state import StateProvider
+from typing import Optional
+
+
+def state_export(
+    cmd,
+    state_file: str,
+    hub_name: Optional[str] = None,
+    resource_group_name: Optional[str] = None,
+    hub_aspects=None,
+    login: Optional[str] = None,
+    auth_type_dataplane: Optional[str] = None,
+    replace: Optional[bool] = False
```

Comment:
nit, since False is explicitly set for replace does Optional add value here?

Reply:
Going to change this to just be bool